api/src/controllers/person.js

```diff
@@ -116,7 +116,8 @@ router.delete(
     const dir = personDocumentBasedir(req.user.organisation, req.params.id);
     const file = path.join(dir, req.params.filename);
     if (!fs.existsSync(file)) {
-      res.status(404).send({ ok: false, message: "File not found" });
+      // if it is not found, it might not be deleted but it's lst in the database, so we return ok.
```

Comment:
Ma contribution (ou alors je sais pas ce que veut dire `lst` ou alors c'est `still`)

```suggestion
      // if it is not found, it might not be deleted but it's still in the database, so we return ok.
```